tests/optimagic/optimization/test_history_collection.py

```diff
@@ -33,12 +33,18 @@ def test_history_collection_with_parallelization(algorithm, tmp_path):
 
     path = tmp_path / "log.db"
 
+    algo_options = {"n_cores": 2}
+    if algorithm == "nevergrad_pso":
+        algo_options["stopping_maxfun"] = 15
+    else:
+        algo_options["stopping_maxiter"] = 3
+
     collected_hist = minimize(
         fun=mark.least_squares(lambda x: x),
         params=np.arange(5),
         algorithm=algorithm,
         bounds=Bounds(lower=lb, upper=ub),
-        algo_options={"n_cores": 2, "stopping_maxiter": 3, "stopping_maxfun": 6},
+        algo_options=algo_options,
```

Comment:
The current version raises warnings because the algorithms over which we loop in this test only have one of the stopping options. I would prefer a warning-free pytest output.